src/App.jsx

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
-import logo from './logo.svg';
+// import logo from './logo.svg';
+import { Route, Switch, useLocation } from 'react-router-dom';
 import './App.css';
+import Home from './pages/Home';
+import Forgot from './pages/Forgot';
```

Comment:
[same as route] we can probably afford to make this `ForgotPassword` instead of `Forgot` to make it more descriptive of what it does